README.md

```diff
@@ -97,15 +115,77 @@ Conjur::API.new_from_key login, api_key
 Note that if you are connecting as a [Host](http://developer.conjur.net/reference/services/directory/host), the login should be 
 prefixed with `host/`. For example: `host/myhost.example.com`, not just `myhost.example.com`.
 
-## Contributing
+# Development
+
+The file `docker-compose.yml` is a self-contained development environment for the project.
+
+## Starting
+
+To bring it up, run:
+
+```sh-session
+$ docker-compose up -d pg conjur_4 conjur_5
+```
+
+Then configure the v4 and v5 servers:
```

Comment:
Can we automate this step by running these scripts as part of the Docker entrypoint or something? Or are there cases where we might not want to configure the containers right away?

Reply:
I could wrap it all in a dev.sh script that’s probably about the best I could do.